web/modules/custom/react_tools/tools/config/webpack.config.prod.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
+const eslintFormatter = require('react-dev-utils/eslintFormatter');
+const paths = require('./paths');
+const getClientEnvironment = require('./env');
+const apps = require('./apps');
+
+const publicPath = paths.servedPath;
+const shouldUseRelativeAssetPaths = publicPath === './';
+const shouldUseSourceMap = false;
+const publicUrl = publicPath.slice(0, -1);
+const env = getClientEnvironment(publicUrl);
+
+if (env.stringified['process.env'].NODE_ENV !== '"production"') {
+  throw new Error('Production builds must have NODE_ENV=production.');
+}
+
+const cssFilename = 'static/css/[name].min.css';
+const extractTextPluginOptions = shouldUseRelativeAssetPaths
+  ? // Making sure that the publicPath goes back to to build folder.
+    { publicPath: Array(cssFilename.split('/').length).join('../') }
+  : {};
+
+module.exports = {
+  bail: true,
+  devtool: shouldUseSourceMap ? 'source-map' : false,
+  entry: apps.entryPoints,
+  output: {
+    path: paths.appBuild,
+    filename: 'static/js/[name].min.js',
+    chunkFilename: 'static/js/[name].chunk.js',
+    publicPath: publicPath
+  },
+  resolve: {
+    modules: ['node_modules'],
+    extensions: ['.web.js', '.mjs', '.js', '.json', '.web.jsx', '.jsx'],
+    alias: {
+      'react-native': 'react-native-web',
+    }
+  },
+  module: {
+    strictExportPresence: true,
+    rules: [
+      // TODO: Disable require.ensure as it's not a standard language feature.
+      {
+        test: /\.(js|jsx|mjs)$/,
```

Comment:
should we add this exclude line? "exclude: /node_modules/,", as far as I understand, this rule is for the linter only. I could be wrong.

Reply:
I am not a webpack expert et, so I am not sure why it fails, but I had to add this as the babel loader was throwing an error. When I googled this error most people said to just exclude node modules. I think it's used for linting as well, but it doesn't just throwing a warning, it throws an error. So this needs to be in for now until we figure out if there is a better solution.

Reply:
Yeah, I saw some examples that were excluding 'node_modules', I assume it's a performance thing. I'm ok with merging this and cleaning it up and improving it little by little.